core/src/main/java/de/valtech/aecu/core/groovy/console/bindings/actions/resource/CreateLabel.java

```diff
@@ -0,0 +1,79 @@
+package de.valtech.aecu.core.groovy.console.bindings.actions.resource;
+
+import de.valtech.aecu.api.service.AecuException;
+import de.valtech.aecu.core.groovy.console.bindings.actions.Action;
+import org.apache.sling.api.resource.PersistenceException;
+import org.apache.sling.api.resource.Resource;
+
+import javax.annotation.Nonnull;
+import javax.jcr.Node;
+import javax.jcr.RepositoryException;
+import javax.jcr.Session;
+
+/**
+ *@author Ian Smets
+ *@author Wietse Vandeput
+**/
+public class CreateLabel implements Action {
+    private final String language;
+    private final String key;
+    private final String value;
+
+    public CreateLabel(String language, String key, String value){
+        this.language = language;
+        this.key = key;
+        this.value = value;
+    }
+
+    @Override
+    public String doAction(@Nonnull Resource resource) throws PersistenceException, AecuException {
+        Session session = resource.getResourceResolver().adaptTo(Session.class);
+        try {
+            Node i18nNode = session.getNode(resource.getPath());
+            Node node = getLanguageNode(i18nNode, language);
+            if (node != null) {
+                try {
+                    String PROPERTY_SLING_MESSAGE = "sling:message";
```

Comment:
create const in class

Reply:
put the const back in the class instead of the method